codecs/vp9_packet.go

```diff
@@ -313,24 +319,30 @@ func (p *VP9Packet) parseRefIndices(packet []byte, pos int) (int, error) {
 
 // Scalability structure (SS):
 //
-//      +-+-+-+-+-+-+-+-+
+//	+-+-+-+-+-+-+-+-+
+//
 // V:   | N_S |Y|G|-|-|-|
-//      +-+-+-+-+-+-+-+-+              -|
+//
+//	+-+-+-+-+-+-+-+-+              -|
+//
 // Y:   |     WIDTH     | (OPTIONAL)    .
-//      +               +               .
-//      |               | (OPTIONAL)    .
-//      +-+-+-+-+-+-+-+-+               . N_S + 1 times
-//      |     HEIGHT    | (OPTIONAL)    .
-//      +               +               .
-//      |               | (OPTIONAL)    .
-//      +-+-+-+-+-+-+-+-+              -|
+//   - +               .
+//     |               | (OPTIONAL)    .
+//     +-+-+-+-+-+-+-+-+               . N_S + 1 times
+//     |     HEIGHT    | (OPTIONAL)    .
+//   - +               .
+//     |               | (OPTIONAL)    .
+//     +-+-+-+-+-+-+-+-+              -|
+//
 // G:   |      N_G      | (OPTIONAL)
-//      +-+-+-+-+-+-+-+-+                           -|
+//
+//	+-+-+-+-+-+-+-+-+                           -|
+//
 // N_G: |  T  |U| R |-|-| (OPTIONAL)                 .
-//      +-+-+-+-+-+-+-+-+              -|            . N_G times
-//      |    P_DIFF     | (OPTIONAL)    . R times    .
-//      +-+-+-+-+-+-+-+-+              -|           -|
 //
+//	+-+-+-+-+-+-+-+-+              -|            . N_G times
+//	|    P_DIFF     | (OPTIONAL)    . R times    .
+//	+-+-+-+-+-+-+-+-+              -|           -|
```

Comment:
Are all of these diagram formatting changes intentional? Some (like the added `//` lines) are inconsistent with the diagrames in the rest of the file.

Reply:
@tmatth that's just the result of `gofmt -w -s`, comments should be inserted in another way in order to avoid conflicts with standard tools (for instance, by using long comments with /* */ )

Reply:
@aler9 ok since it landed in v1 already it'd make more sense to redo the diagrams with `/* */` in a subsequent PR then.